lightwood/analysis/explain.py

```diff
@@ -223,7 +223,7 @@ def explain(data: pd.DataFrame,
         insights['prediction'] = insights['prediction'].astype(int)
         insights['upper'] = insights['upper'].astype(int)
         insights['lower'] = insights['lower'].astype(int)
-    elif target_dtype in (dtype.float):
+    elif target_dtype in (dtype.float, dtype.quantity):
```

Comment:
im ok with quant being a float but what sort of things are taken as a quant INSTEAD of float/int in use cases?